### DISCUSSION TITLE
Testing question involving known issue with JAWS affecting PowerPoint

### DISCUSSION THREAD

**Discussion by Syracopoulos** [gc-da11yn/gc-da11yn.github.io#578] [Content (Open Discussion)]
Question for the hive brain. This thread was brought to our attention from a colleague, who read it on the IAAP Connections Community forum. I am submitting the conversation to open a discussion about whether this information should be on the DAT in some capacity. It is a known issue, just wondering if we should mention it for people testing their PPT presentations.

Hi, My name is Prachi,
When I work on PPT accessibility, I sometimes encounter issues where the reading order appears correct through NVDA, but JAWS does not follow the proper reading order. I've checked through the slide master but haven't found any issues. Is there a known issue with the JAWS screen reader when testing PowerPoint accessibility?
I am aware of the reading order issue you describe, and to the best of my knowledge it is a bug that is not fixable. It occurs on slides that contain images and text. JAWS read the slide title, then the heading (if there is one), then the Alt Text for the images, then all the text, regardless of how you set the reading order. By contrast, NVDA follows the specified reading order. I have seen this many times and have created test slides to verify the behaviour. I have been unable to find a fix or a viable workaround. The same issue occurs if the slide contains a chart, and possibly other types of content.
One workaround is to mark all images as decorative. If they had Alt Text, put it in a text placeholder and hide it behind the image. Then set the reading order as required. It's a bit more work, but it fixes the problem. However, "ordinary users" hate doing this and probably go back to using Alt Text when I'm not looking.

Hi Prachi
This is a known issue with JAWS. Freedom Scientific are aware of it and it is on their backlog to fix but apparently not that many people have complained about it, so it's low priority. What often happens is that JAWS reads out all links and images on the slide first before reading the other items from the reading order. I find it really annoying so I tend to use Windows Narrator now for checking PowerPoint slides. We've also sent out comms to all our JAWS/Fusion users to let them know about the issue.

**Comment by shawnthompson** [comment]
I don't think this should be on our site. There are so many bugs with screen reader and application combinations like this that we would be having to keep this information up to date and continuously be checking to see if the companies have fixed the bugs. Although there might be websites that list know accessibility issues like this and we could link to them if we know of any.

[Accessibility Support](https://a11ysupport.io/) is one that might be useful. Having said that, I would like us to contribute to these sites if we can with any bugs we find. Giving back to the community and breaking down the silos. 

What are your thoughts @JulieBB23 ?

**Reply by JulieBB23** [reply]
I agree with you Shawn, we shouldn't have that on our website, but we should reply to them and redirect them to the right places. 
As for contribution to these site, i think by we should suggest to our "users" to submit their issue themselves...
